src/node/blockstorage.cpp

```diff
@@ -224,8 +216,8 @@ bool BlockManager::LoadBlockIndex(
     // Calculate nChainWork
     std::vector<std::pair<int, CBlockIndex*>> vSortedByHeight;
     vSortedByHeight.reserve(m_block_index.size());
-    for (const std::pair<const uint256, CBlockIndex*>& item : m_block_index) {
-        CBlockIndex* pindex = item.second;
+    for (auto& [_, block_index] : m_block_index) {
+        CBlockIndex* pindex = &block_index;
         vSortedByHeight.push_back(std::make_pair(pindex->nHeight, pindex));
```

Comment:
nit in the for-loop commit: Any reason to keep this alias?

```suggestion
        vSortedByHeight.emplace_back(block_index.nHeight, &block_index));
```